### ISSUE TITLE
Japanese (I think) characters in Keyboard Mapping screen

### ISSUE THREAD

**Issue by rahlhar** [slajerek/RetroDebugger#70]
After a fresh install (or deleting `~/.RetroDebugger`) the `Settings > C64 > Keyboard mapping` screen appears as expected.

After manipulating the workspace layout - can't think of anything else that I've done in the interrim - the mapping screen shows japanese glyphs in the mapping screen.

**To Reproduce**
Not totally certain

**Expected behavior**
Keyboard mapping screen to remain as default.

**Screenshot**
![0_c64keys](https://github.com/user-attachments/assets/6aa7a199-5ebf-4a2d-9abc-db0fedb289b8)

**Desktop:**
 - OS: Artix Linux
```sh
$ uname -srmo
Linux 6.10.12-hardened1-1-hardened x86_64 GNU/Linux
```

**Comment by rahlhar** [comment]
Well that was foolish of me.
I just grabbed the "highest looking" version number from `vice`'s files, in this case `chargen-906143-02.bin`.
I'm sure it worked at first, but perhaps it was prior to a restart.
I've switched to `chargen-901225-01.bin` and all seems to be behaving.

Thanks again.

Is there a way to know more about these different ROM versions?
I was using [this](https://vice-emu.sourceforge.io/vice_3.html) as reference but it lacks some detail.

Or are there other sources of these ROMs?
